jbpm-flow/src/main/java/org/jbpm/process/core/timer/impl/QuartzSchedulerService.java

```diff
@@ -169,7 +169,8 @@ public void internalSchedule(TimerJobInstance timerJobInstance) {
 
         // Define a Trigger that will fire "now"
         org.quartz.Trigger triggerq = new SimpleTrigger(quartzJobHandle.getJobName()+"_trigger", quartzJobHandle.getJobGroup(), nextFireTime);
-            
+        logger.debug("triggerq.name = " + triggerq.getName() + ",  triggerq.startTime = " + triggerq.getStartTime()); // nextFireTime is mapped to startTime
```

Comment:
@tkobayas could you please use "triggerq.name = {}" syntax instead of string concat?